host/traceh.c

```diff
@@ -365,6 +365,93 @@ oe_result_t oe_log_set_callback(void* context, oe_log_callback_t callback)
     return OE_UNEXPECTED;
 }
 
+void oe_log_dump_message(
```

Comment:
I don't see the need on introducing both `oe_log_dump_message` and `oe_log_dump`. Can you double check if they are really necessary?

Reply:
OE_TRACE come with the set formatting :
```
2022-08-25T22:42:15+0000.021630Z [(E)INFO] tid(0x7ff51fe80f40) | enclave.signed:claim attributes[8]:  [/home/ratrames/dev/openenclave/common/sgx/verifier.c:oe_sgx_verify_evidence:944]
2022-08-25T22:42:15+0000.021638Z [(E)INFO] tid(0x7ff51fe80f40) | enclave.signed:0300000000000000
```

I can use OE_TRACE instead of introducing oe_log_dump if that is okay

Reply:
Another problem would be, when the data is longer than OE_LOG_MESSAGE_LEN_MAX, the data might be truncated because we the header `2022-08-25T22:42:15+0000.021638Z [(E)INFO] tid(0x7ff51fe80f40) | enclave.signed:`  is not of fixed length.